lib/pry/repl.rb

```diff
@@ -105,9 +105,16 @@ def read
         indented_val = @indent.indent(val)
 
         if output.tty? && pry.config.correct_indent && Pry::Helpers::BaseHelpers.use_ansi_codes?
+          suffix_whites = if readline_available? && Readline.vi_editing_mode?
```

Comment:
I don't quite understand the name of this variable. Could you elaborate or change it to something more meaningful? Perhaps, `overhang` since `correct_indentation`'s 3rd parameter is named `overhang`?

Reply:
The reasoning was "how many __white__ spaces to __suffix__" but sure, will rename to `overhang`